### ISSUE TITLE
Invalid json packets are not validated during api request 

### ISSUE THREAD

**Issue by vtf05** [a85/POSTMan-Chrome-Extension#11821]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

While I was making an api call through postman I got to know that even i have a invalid json packet postman allows it to be send through api , though it is identified by postman as invalid packet but still allowing me to send this packet 
<img width="1014" alt="Screenshot 2023-03-15 at 11 38 51 AM" src="https://user-images.githubusercontent.com/56218201/225221754-5758e54d-a555-48c1-bee6-cad1d8bba5f7.png">

### Steps To Reproduce

{
    "address":
}
use this kind of a packet to make a post api call 

### Screenshots or Videos

<img width="1005" alt="Screenshot 2023-03-15 at 11 39 36 AM" src="https://user-images.githubusercontent.com/56218201/225221875-39f3744d-5f52-41ff-97f4-cf16d7a7d5e3.png">

### Operating System

macOs

### Postman Version

9.21.5

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by giridharvc7** [comment]
@vtf05 yes, we don't take a opinion on this and block users from request sending. 
What is an ideal solution you are looking for here?

**Comment by vtf05** [comment]
ideally if the packet is invalid it should be blocked to send a request
